docs/docs/hpc-on-k8s/minio.md

```diff
@@ -0,0 +1,25 @@
+# Deploying Minio in Kubernetes
+
+To deploy Minio in our Kubernetes cluster, we utilize Helm charts. Prior to installation, it's crucial to ensure that the Helm CLI is properly set up and the kubeconfig file is located in the designated directory.
+
+For this installation, we've opted for the default namespace, employing the following configurations:
+- storageClass:  
```

Comment:
There needs to be an empty line above here otherwise the bullet list doesn't render properly

Reply:
updated and image is present in openflighthpc dockerhub account.